reV/handlers/outputs.py

```diff
@@ -522,8 +522,13 @@ def _create_dset(self, ds_name, shape, dtype, chunks=None, attrs=None,
             Dataset attributes
         data : ndarray
             Dataset data array
+        replace : bool
+            If previous dataset exists with the same name, it will be replaced.
         """
         if self.writable:
+            if ds_name in self.datasets and replace:
+                del self.h5[ds_name]
+
```

Comment:
need an elif that skips the create_dataset step.

Reply:
Previous behavior was to break on h5py "link already exists" error. I think this is important if the shape and dtype don't match. I'm adding a new commit that checks for this.

Reply:
I'm surprised this logic works, the else won't be reached after you delete the dset so you won't re-create the dataset

Reply:
Yep you're right. fixed.